solutions/src/test/java/ru/sberbank/school/task13/cache/Temp.java

```diff
@@ -0,0 +1,30 @@
+package ru.sberbank.school.task13.cache;
+
+import java.io.Serializable;
+
+/**
+ * Created by Mart
+ * 20.07.2019
+ **/
+public class Temp implements Serializable {
+    private String superTemp;
+
+    public Temp(Double superTemp) {
+        this.superTemp = superTemp.toString();
+    }
+
+    public String getSuperTemp() {
+        return superTemp;
+    }
+
+    public void setSuperTemp(Double superTemp) {
+        this.superTemp = superTemp.toString();
+    }
+
+    @Override
+    public String toString() {
+        return "Temp{" +
+                "superTemp='" + superTemp + '\'' +
```

Comment:
&#39;+&#39; should be on a new line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-325087852>*